src/client/components/pages/Courses/MeetingTimesList.tsx

```diff
@@ -317,6 +322,22 @@ export const MeetingTimesList
                           Room:
                           {currentEditMeeting.room
                         && currentEditMeeting.room.name}
+                          {currentEditMeeting.room && (
+                            <BorderlessButton
+                              alt={`Remove Room ${index + 1} on ${meetingTimeString}${meetingRoomString}`}
```

Comment:
I'd suggest using the room name instead of just it's index. If someone is using a screen reader they'll hear "Remove room 1" which isn't terribly intuitive. [Jon and I talked about this in another PR](https://github.com/seas-computing/course-planner/pull/361#discussion_r669149001) and we ended up changing the alt text to be a little more verbose.

The solution Jon ended up with was

```js
 alt={`Delete Meeting ${index + 1} on ${meetingTimeString}${meetingRoomString}`}
```
You can see that in context [here](https://github.com/seas-computing/course-planner/pull/361/files#diff-6334f618ea28756872cb5098b9cbc84f0f1a0b690c0d2b7a74f0481812981890R182)

Reply:
I think I may be misunderstanding something - the alt text for the remove room button includes the time and room info as well:
![image](https://user-images.githubusercontent.com/29589689/126808598-733b034c-af26-487c-a33e-598dca30fef0.png)
Both the delete meeting and remove room button's alt text include `${meetingTimeString}${meetingRoomString}`

Reply:
Ah, yes of course. Didn't spot that at first! Sorry.